lib/cipher.js

```diff
@@ -45,24 +45,23 @@ class TuyaCipher {
   * returns object if data is JSON, else returns string
   */
   decrypt(data) {
+    // Incoming data format
+    let format = 'buffer';
+
     if (data.indexOf(this.version.toString()) !== -1) {
```

Comment:
Although not part of your recent changes, I noticed this during the review. The version string should always be the first 3 characters, so I recommend `if (data.startsWith(this.version)) {`

Reply:
Yes, but the function accepts Buffers as well as strings. Buffers don't have a `.startsWith()` function.  Updated the documentation to make that more clear.

Reply:
Ah yes, didn't notice it took Buffers as well. In that case, it should be `if (data.indexOf(this.version.toString()) === 0) {`.